server/index.js

```diff
@@ -19,6 +19,17 @@ app.get('/api/health-check', (req, res, next) => {
     .catch(err => next(err));
 });
 
+const viewAllProducts = `
```

Comment:
This is great! My only suggestion would be to put this const inside of the app.get function, since you will just be using it for this api call.